spec/helpers/application_helper_spec.rb

```diff
@@ -1641,4 +1641,23 @@
       ).to eq('host_services')
     end
   end
+
+  describe "#multiple_relationship_link" do
+    context "When record is a Container Provider" do
```

Comment:
> how GET of ems_container/1 will reach the show action?

@abonas in routes.rb [1] we use the `resources` [2] keyword that routes to 'show' action.

[1] https://github.com/ManageIQ/manageiq/blob/master/config/routes.rb#L2650
[2] http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions